app/boards/intel_adsp_cavs25.conf

```diff
@@ -7,6 +7,10 @@ CONFIG_AMS=y
 CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=30
+CONFIG_MM_DRV=y
+CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM=y
+CONFIG_INTEL_MODULES=y
+CONFIG_LIBRARY_MANAGER=y
```

Comment:
I guess we can load libraries on TGL. But can we use them there? AFAICS to use a library we need to map it and I'm not sure this is supported on TGL? We do seem to have a non-MTL driver drivers/mm/mm_drv_intel_adsp_tlb.c for that but I'm not actually sure if it's for TGL and whether it was ever tested with library loading

Reply:
Hrm, yes, I don't know, these are needed to actually make the loading to work on TGL family. I guess we need to start from somewhere to get the loading to work if we want to?

Reply:
I'm ok to go with incremental change, but it's surprising to see this works as the TLB_REMAP driver in Zephyr is not available to cAVS2.5.

Reply:
Right, I will drop the CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM and add two more patches to handle the Kconfig variations better.